src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -217,7 +216,7 @@ void GcsResourceManager::HandleGetAllResourceUsage(
     for (const auto &demand : aggregate_load) {
       auto demand_proto = batch.mutable_resource_load_by_shape()->add_resource_demands();
       demand_proto->CopyFrom(demand.second);
-      for (const auto &resource_pair : demand.first) {
+      for (const auto &resource_pair : demand.first.shape) {
```

Comment:
Is it safe to drop the label selector here? Shouldn't we carry those label selectors to the response?